client/src/components/DisplayTable/DisplayTableUtils.scss

```diff
@@ -13,4 +13,7 @@ button.display_table__util-icon {
   &:active {
     transform: scale(1.15);
   }
+}
+.no-right {
```

Comment:
Oh, later on I make a comment about utility classes that would apply to this too. They should be defined globally.